generate-errors/Cargo.toml

```
@@ -8,10 +8,10 @@ authors = [
 license = "MIT"
 edition = "2018"
```

```suggestion
edition = "2021"
```

This doesn't change anything in the code, but it is probably good to have. ([Rust 2021 migration guide](https://doc.rust-lang.org/edition-guide/rust-2021/index.html))